bin/assigner/CMakeLists.txt

```diff
@@ -34,8 +34,12 @@ elseif(CMAKE_CROSSCOMPILING)
     endif()
 endif()
 
+string(TOUPPER ${CURRENT_PROJECT_NAME} UPPER_CURRENT_PROJECT_NAME)
+
 cm_setup_version(VERSION 0.1.0)
```

Comment:
The overall design looks good, but the problem is that the version is set statically here (actually, my guess is that it came from an old code after copy-pasting the CMakeLists's content). As I see it, it makes sense to use `git describe` - since that's how we set the version in the release action. There is already such code for obtaining  zkLLVM version (which is basically equal to the `assigner` version)

Reply:
OK, i'll try to use `git describe` here.
Actually we neet to do the same for `tranpiler`